rep-0003.rst

```diff
@@ -250,6 +310,13 @@ consulting Ubuntu's release and end-of-life schedule [1]_.
 
 These targets, including starting and ending support dates, are based on the Distribution Timeline to meet minimum requirements. [3]_
 
+Architectures
+-------------
+
+As of ROS Lunar, we do not build packages for x86 architectures.
```

Comment:
Yeah, debian packaging calls it i386 since amd64 is also known as x86_64. (Despite the fact that it's actually an i686 or something along those lines, it's in the i386-architecture.[1](https://askubuntu.com/questions/54296/difference-between-the-i386-download-and-the-amd64) I'd suggest we follow the debian practice and refer to it at i386.